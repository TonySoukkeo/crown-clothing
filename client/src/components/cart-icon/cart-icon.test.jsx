import React from "react";
import { shallow, configure } from "enzyme";
import CartIcon from "./cart-icon.component";

import Adapter from "enzyme-adapter-react-16";

configure({
  adapter: new Adapter(),
});

let wrapper;

beforeEach(() => {
  wrapper = shallow(<CartIcon />);
});

it("renders cart-icon component without crashing", () => {
  expect.assertions(1);
  expect(wrapper).toMatchSnapshot();
});
