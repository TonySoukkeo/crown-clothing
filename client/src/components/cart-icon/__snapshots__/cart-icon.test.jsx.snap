// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders cart-icon component without crashing 1`] = `ShallowWrapper {}`;
